ccnet

```diff
@@ -1,42 +1,104 @@
-#!/bin/bash
+#!/bin/python3
```

Comment:
This doesn't work in mac. Can you try and see if `#!/usr/bin/env python3` works in linux? If so, let's move it to that

Reply:
Ya it does

Reply:
Changed it now